lib/protobuf/field/field_array.rb

```diff
@@ -65,7 +65,7 @@ def normalize(value)
         fail TypeError, "Unacceptable value #{value} for field #{field.name} of type #{field.type_class}" unless field.acceptable?(value)
 
         if field.is_a?(::Protobuf::Field::EnumField)
-          field.type_class.fetch(value)
+          fetch_enum(field.type_class, value)
```

Comment:
Doesn't look like we have any tests around this change. I'd say it would be better to remove it from this patch if it's not needed for map type support. It's a good idea, but I'm having some difficulty understanding if our code is going to be impacted by this. That's a lame excuse, I know, but I think it would better to get map types up first and then we can look into changing preconditions.